package.json

```diff
@@ -33,16 +35,18 @@
     "sweetalert": "^2.1.2",
     "vite": "^5.0.12",
     "vite-tsconfig-paths": "^4.3.1",
+    "vitest": "^1.2.2",
```

Comment:
That is a topic for another task, am I right?

Reply:
removed